client/src/components/CreateContestForm.js

```diff
@@ -104,12 +104,34 @@ export default function CreateContestForm() {
             }
         })
     }
-
-    // temporary hardcode image names
-    const imageNames = ['tattoo-1.png', 'tattoo-2.png', 'tattoo-3.png', 'tattoo-4.png', 'tattoo-5.png', 
-    'tattoo-6.png', 'tattoo-7.png', 'tattoo-8.png', 'tattoo-9.png', 'tattoo-10.png', 
-    'tattoo-11.png', 'tattoo-12.png', 'tattoo-13.png', 'tattoo-14.png']
-
+    useEffect(() => { // Only runs once when first rendering
+        getInspirationalImages((data) => {
```

Comment:
```suggestion
        getInspirationalImages(createGridListTiles, (error) => {
        ...
```

just a more simple and shorter way of writing this out